blockonomics-woocommerce.php

```diff
@@ -411,15 +411,29 @@ function bnomics_display_tx_info($order, $email=false)
         $blockonomics = new Blockonomics();
         $active_cryptos = $blockonomics->getActiveCurrencies();
         foreach ($active_cryptos as $crypto) {
-            $txid = get_post_meta($order->get_id(), 'blockonomics_'.$crypto['code'].'_txid', true);
-            $address = get_post_meta($order->get_id(), $crypto['code'].'_address', true);
+            $txid = get_post_meta($order->get_id(), 'blockonomics_payments_txids', true);
```

Comment:
`blockonomics_payments_txids` and `blockonomics_payments_addresses` now holds both Bitcoin and Bitcoin Cash addresses and transaction id's. We can remove 
```
$active_cryptos = $blockonomics->getActiveCurrencies();
foreach ($active_cryptos as $crypto) {
```
and update `if ($crypto['code'] == 'btc') {` to check if the address starts with `q` or `p` to indicate it is a Bitcoin Cash address.

Reply:
I have made the changes as requested. Let me know what you think.